com.salesforce.b2eclipse.tests/projects/build-with-subpackage/module1/BUILD

```diff
@@ -0,0 +1,25 @@
+load("@rules_java//java:defs.bzl", "java_binary")
+load("@salesforce_rules_mybuilder//mybuilder:defs.bzl", "mybuilder_gen")
```

Comment:
it would be better to test on very specific, distilled test projects.
tmm, potential break or change in salesforce_rules_mybuilder project shouldn't cause test failure.
at the same time, work of this specific source generator may be a subject for separate test